src/services/quiz.service.ts

```diff
@@ -49,12 +54,34 @@ export async function copyQuizBankToClassroom(token: string, quizbankId: string,
       Authorization: `Bearer ${token}`,
     },
   }
-  return fetch(url, options)
-    .then(async(res) => {
-      if (!res?.ok) {
-        throw new Error(res.statusText)
-      }
-      return true
-    })
+  return fetch(url, options).then(async (res) => {
+    if (!res?.ok) {
+      throw new Error(res.statusText)
+    }
+    return true
+  })
 }
 
+export async function deleteQuizBank(token: string, quizbankId: string) {
+  const url = getAPIServerURL(`/api/quizbank/${quizbankId}`)
+
+  const options = {
+    method: "DELETE",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${token}`,
+    },
+  }
+  return fetch(url, options).then(async (res) => {
+    if (!res?.ok) {
+      return {
+        isSuccess: false,
+        message: res.statusText,
+      }
+    }
+    console.log(res.json())
+    return {
+      isSuccess: true,
+    }
+  })
+}
```

Comment:
Separate this into a action.  and following the pattern of fetch server (refer to `src\app\[locale]\(main)\quizbank\actions\get-my-quizbank-action.tsx`